app/src/main/java/com/codesoom/assignment/App.java

```diff
@@ -1,10 +1,11 @@
 package com.codesoom.assignment;
 
 public class App {
-    public String getGreeting() {
+    public static String getGreeting() {
         return "Hello World!";
     }
 
     public static void main(String[] args) {
+        System.out.println(getGreeting());
```

Comment:
static은 최소한으로 쓰는 게 좋습니다.

```suggestion
        App app = new App();
        System.out.println(app.getGreeting());
```

Reply:
네 감사합니다!